ci/k8s/run-chaos-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CHAOS_FAULT="${1:-no_fault}"
+CHAOS_WORKLOAD="${2:-test_potato_passer}"
+
+bazel run //ci/ray_ci:build_in_docker -- docker --python-version 3.8 \
+    --platform cpu --canonical-tag kuberay-test
+docker tag rayproject/ray:kuberay-test ray-ci:kuberay-test
+
+bash python/ray/tests/chaos/prepare_env.sh
+
+if [[ "${CHAOS_FAULT}" != "no_fault" ]]; then
+    kubectl apply -f "python/ray/tests/chaos/${CHAOS_FAULT}.yaml"
+fi
+
+echo "--- Prepare the test container."
+docker run -d --network host --name chaos-test ray-ci:kuberay-test sleep infinity
+docker exec -ti chaos-test mkdir -p /tmp/rayci/python/ray/tests/chaos
```

Comment:
does the script have to run inside this folder path? what if just copy the files in to /tmp/rayci

Reply:
the script's content has assumption that is runs at the root. it is probably very easy to change, but I do not want to change it here.

if you feel strongly, I can do it in a follow up.

only files under the directory is copied in, so the dependency is well restricted.